test/prometheus_test.go

```diff
@@ -0,0 +1,139 @@
+package test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/blang/semver"
+	"github.com/google/uuid"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/kind/pkg/apis/config/v1alpha3"
+	"sigs.k8s.io/kind/pkg/cluster"
+
+	"github.com/mesosphere/kubeaddons/pkg/test"
+	"github.com/mesosphere/kubeaddons/pkg/test/cluster/kind"
+	"github.com/mesosphere/kubeaddons/pkg/test/loadable"
+)
+
+func TestPrometheusUp(t *testing.T) {
```

Comment:
Having this as a `Test$_(t *testing.T)` will make it run on all CI runs, which wont be right for updates to other addons not related. We should instead make sure these tests land under the `TestPrometheusGroup()` test function.

We should model it similar to the [thanos tests](https://github.com/mesosphere/kubeaddons-kommander/blob/master/test/thanos_test.go#L12) in kommander, wherein the tests are [loaded into the test harness](https://github.com/mesosphere/kubeaddons-kommander/blob/master/test/kommander_test.go#L62) to run as part of the default (post-deployment, pre-cleanup) test plan.

It may be valid to simply rename this and remove the old one:

```suggestion
func TestPrometheusGroup(t *testing.T) {
```
However if we do that we need to make sure we handle the other members of the group the same was as was done in the previous function.